index.html

```diff
@@ -264,8 +273,8 @@ <h2 id="project_tagline"></h2>
 </code></pre>
 
 <p><strong>NOTE:</strong> For other than eiffelsemantics protocol, provide the Java opts as: </p>
-<pre>set JAVA_OPTS="-Djava.ext.dirs=/path/to/other/protocol/jars" in catalina.sh</pre>
-
+<pre>set JAVA_OPTS="-Djava.ext.dirs=/usr/home/jars/" in catalina.sh</pre>
```

Comment:
Isn't it better to have an example path like "/path/to/jars" to make it obvious the user have to provide there own path?